renku/api/repository.py

```diff
@@ -99,11 +100,13 @@ def __attrs_post_init__(self):
 
         # initialize submodules
         if self.repo:
-            check_output([
-                'git', 'submodule', 'update', '--init', '--recursive'
-            ],
-                         cwd=str(self.path))
-        # TODO except
+            try:
+                check_output([
+                    'git', 'submodule', 'update', '--init', '--recursive'
+                ],
+                             cwd=str(self.path))
+            except subprocess.CalledProcessError:
+                pass
```

Comment:
why fail silently? We should do/say _something_

Reply:
We are not failing silently. Git output will be shown.